fourmat/assets/.isort.cfg

```diff
@@ -2,3 +2,4 @@
 profile = black
 combine_as_imports = True
 line_length = 79
+src_paths = .
```

Comment:
uh. this was added in https://github.com/4Catalyzer/fourmat/pull/20/files#diff-170b3556059bb1e212db57e7c98d02b6R5 how come it shows as added here?

Reply:
i'm an idiot and i added it in the wrong place